src/gromacs/nblib/bondtypes.h

```diff
@@ -72,12 +72,17 @@ class HarmonicBondType
 
     EquilDistance equilDistance() const { return equilDistance_; }
 
+    static auto getProperties();
```

Comment:
I am not convinced with the name `getProperties`. Should we just say `properties`?

Reply:
I'd second calling it `properties()` too.

Reply:
done